ci/test/lint-main/checks/check-bazel.sh

```diff
@@ -18,7 +18,15 @@ cd "$(dirname "$0")/../../../.."
 . misc/shlib/shlib.bash
 
 if [[ ! "${MZDEV_NO_BAZEL_CHECK:-}" ]]; then
-  try bin/bazel gen
+  if try bin/bazel gen; then
+      :
```

Comment:
This is a NOP, right?

Reply:
Yes. The reason it's required is because doing `if ! try bin/bazel gen` makes `!` the last command, and thus `$?` is 0.